### ISSUE TITLE
Error when using only chart and pivotchart

### ISSUE THREAD

**Issue by JeroenVinke** [aurelia-ui-toolkits/aurelia-syncfusion-bridge#68]
Hey, I'm trying to only use the pivotchart and the cart, but when I do that I get an error:

![image](https://user-images.githubusercontent.com/2189477/28866296-cc9bbca8-7772-11e7-8be1-059a2a8b4a6a.png)

where that line is:

![image](https://user-images.githubusercontent.com/2189477/28866376-16998632-7773-11e7-919d-6678c368f460.png)

**aurelia.json:**
```

          {
            "name": "aurelia-syncfusion-bridge",
            "path": "../node_modules/aurelia-syncfusion-bridge/dist/amd",
            "main": "index",
            "resources": [
              "common/*.js",
              "pivotchart/*.js",
              "chart/*.js"
            ]
          },
          {
            "name": "syncfusion-javascript",
            "path": "../node_modules/syncfusion-javascript",
            "main": false,
            "resources": [
              "Content/ej/web/ej.widgets.core.bootstrap.min.css",
              "Content/ej/web/bootstrap-theme/ej.theme.min.css"
            ]
          },
```

**main.js:**
`.plugin('aurelia-syncfusion-bridge', syncfusion => syncfusion.ejChart().ejPivotChart())`

I was able to work around it by importing the missing file manually from main.ts:

`import 'syncfusion-javascript/Scripts/ej/web/ej.pivotanalysis.base.min';`

That resolved the problem. Could it be that this import statement is missing from either the bridge or Syncfusion?

Let me know if you need more info. Thanks!

**Comment by karthickthangasamy** [comment]
@JeroenVinke Thanks for detailed information.

The issue will be fixed by including `ej.pivotanalysis.base.min` in `ejPivotChart.min.js` source `amd` loader mapping.

I will fix and update the bridge at the earliest.

**Comment by JeroenVinke** [comment]
Thanks for the quick response @karthickthangasamy. No need to hurry, the workaround works fine :+1:

**Comment by karthickthangasamy** [comment]
The issue resolved in aurelia-syncfusion-bridge@1.0.0
